### DISCUSSION TITLE
Add Markdownlint to CI

### DISCUSSION THREAD

**Discussion by dylanhitt** [go-fuego/fuego#121] [General]
What are the feelings about implementing [markdownlint](https://github.com/markdownlint/markdownlint). This could be pretty useful our documentation grows?

Same pattern as other things probably will put a check in the CI to ensure nothing slips through. I'll probably add a target to the `Makefile` called `lint-markdown` then add a catch all lint target that is `lint-all: lint-markdown lint` for local checking.

**Comment by EwenQuim** [comment]
Good idea!

OK for the Makefile target, I think that's a good idea to keep linters and all that way.

**Comment by dylanhitt** [comment]
Completed by #138
